### ISSUE TITLE
auto prune docker images on minikube start

### ISSUE THREAD

**Issue by medyagh** [kubernetes/minikube#7630]
to reduce disk pressure.
(this can be done in parallel one a start)

more info https://blog.tilt.dev/2019/10/28/build-cache-spark-joy.html

it would interesting to find out, would it be bad if we always prune images ? would there be cases that users be mad at us for deleteing abandoned images ?

**Comment by afbjorklund** [comment]
It would be appropriate to at least only do this for the docker driver

**Comment by medyagh** [comment]
> It would be appropriate to at least only do this for the docker driver

why only docker driver ? I meant the docker images inside minikube

**Comment by medyagh** [comment]
still on the backlog.

**Comment by priyawadhwa** [comment]
I think it's safe to assume that users are using all of the images that are in the docker daemon _within_ minikube -- I can't think of a situation where users would want us to prune that daemon for them.

**Comment by medyagh** [comment]
no longer needed
